.github/workflows/deploy.yml

```diff
@@ -24,9 +24,6 @@ jobs:
       - name: Build website
         run: npm run build
         env:
-          ALGOLIA_APP_ID: ${{ secrets.ALGOLIA_APP_ID }}
-          ALGOLIA_SEARCH_API_KEY: ${{ secrets.ALGOLIA_SEARCH_API_KEY }}
-          ALGOLIA_INDEX_NAME: ${{ secrets.ALGOLIA_INDEX_NAME }}
           ENVIRONMENT: production
```

Comment:
you can remove the entire `env` since we don't need to use it anymore

Reply:
done